home_task_1/package.json

```diff
@@ -9,6 +9,7 @@
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-scripts": "5.0.1",
+    "sass": "^1.52.2",
```

Comment:
you probably don't need it in the dependencies. Create-react-app should know how to handle everything